dependency in wrong package list

**Describe the bug**
Repo lists dependency is dev but requires in federation package

**What version are you on?**
14.3.1

**To Reproduce**
install as a dependency in any other project

**Expected behavior**
not to err

**Additional context**
might be doable with https://github.com/stellar/js-stellar-sdk/issues/1260
```sh
@stellar/stellar-sdk v14.3.1 has a packaging bug: urijs is listed as a devDependency but the runtime code (federation/server.js) requires it.

node:internal/modules/cjs/loader:1249
  const err = new Error(message);
              ^

Error: Cannot find module 'urijs'
Require stack:
- node_modules/@stellar/stellar-sdk/lib/federation/server.js
- node_modules/@stellar/stellar-sdk/lib/federation/index.js
- node_modules/@stellar/stellar-sdk/lib/index.js
```

--- a/package.json
+++ b/package.json
@@ -188,8 +188,7 @@
     "taffydb": "^2.7.3",
     "terser-webpack-plugin": "^5.3.14",
     "ts-node": "^10.9.2",
-    "typescript": "5.6.3",
-    "urijs": "^1.19.1",
+    "typescript": "5.6.3",    
     "vitest": "^3.2.4",
     "webpack": "^5.102.1",
     "webpack-cli": "^5.0.1"
@@ -201,7 +200,8 @@
     "eventsource": "^2.0.2",
     "feaxios": "^0.0.23",
     "randombytes": "^2.1.0",
-    "toml": "^3.0.0"
+    "toml": "^3.0.0",
+    "urijs": "^1.19.1"
   },
   "overrides": {
     "chalk": "5.3.0",

